client/my-sites/checkout/upsell-nudge/plan-upgrade-upsell/index.jsx

```diff
@@ -0,0 +1,260 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import Gridicon from 'gridicons';
+import formatCurrency from '@automattic/format-currency';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Button from 'components/button';
+import DocumentHead from 'components/data/document-head';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export class PlanUpgradeUpsell extends PureComponent {
+	render() {
+		const { receiptId, translate } = this.props;
+
+		const title = translate( 'Checkout ‹ Plan Upgrade', {
+			comment: '"Checkout" is the part of the site where a user is preparing to make a purchase.',
+		} );
+
+		return (
+			<>
+				<PageViewTracker path="/checkout/:site/plan-upgrade-nudge/:receipt_id" title={ title } />
```

Comment:
Just caught this via debugging Hypothesis (:pain_face:) 

This should be:
/checkout/:site/offer-plan-upgrade/premium/:receiptId

Probably better to address this once the AB test is done so we don't mess with the data.

cc: @niranjan-uma-shankar